doc/authentication-methods/client-certificate.md

```diff
@@ -28,6 +28,13 @@ A `SASL EXTERNAL` authentication method is disabled by default.
 In order to enable it, please add [`sasl_mechanisms` option](../Advanced-configuration.md#authentication) to MongooseIM config file.
 Its value must include a `cyrsasl_external` item.
 Obviously the list may be longer, if the system should support both certificate and password based authentication.
+`SASL EXTERNAL` authentication method requires the use of digital
+certificates. If the `xmpp_addrs` fields are included in certificate, they are always checked first.
+In case no `xmpp_addrs` are specified, the `cn` (_common name_) field might be
+used, but it is optional. If there are more than one `xmpp_addrs` or the
```

Comment:
"Or" or "on"?
If or:
If there is more than one `xmpp_addrs` or both the list and `cn` field are empty, the client must include the authorization entity.

if on:

If there is more than one `xmpp_addrs` on the list and the`cn` field is empty, the client must include the authorization entity.